app/graphql/types/language_type.rb

```diff
@@ -0,0 +1,9 @@
+module Types
+  class LanguageType < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :email, String, null: false
```

Comment:
Did you accidentally add email to this type? @lkriffell

Reply:
Yes, good catch